Rick-and-Morty/Rick And Morty/Model/Character.swift

```diff
@@ -8,6 +8,19 @@
 
 import Foundation
 
+
+struct CharactersList: Decodable {
+    let info: CharacterListInfo
+    let results: [Character]
+}
+
+struct CharacterListInfo: Decodable {
```

Comment:
This could then become `CharacterResponseInfo` if you go with `CharactersResponse` above